orc8r/cloud/go/services/ctraced/ctraced/main.go

```diff
@@ -50,6 +50,12 @@ func main() {
 	}
 	ctracedBlobstore := ctraced_storage.NewCtracedBlobstore(fact)
 
+	// Init gRPC servicer
+	tracingServicer := servicers.NewCallTraceServer(ctracedBlobstore)
+	protos.RegisterCallTraceControllerServer(srv.GrpcServer, tracingServicer)
```

Comment:
Nit: do these in a single line, then remove the newline between the two servicer registration lines